pkg/operator/agent/agent_test.go

```diff
@@ -64,6 +64,14 @@ func TestStartAgentDaemonset(t *testing.T) {
 	err := a.Start(namespace)
 	assert.Nil(t, err)
 
+	// check clusters rbac roles
+	_, err = clientset.CoreV1().ServiceAccounts(namespace).Get("rook-agent", metav1.GetOptions{})
+	assert.Nil(t, err)
```

Comment:
do we want to spot check some of the specific permissions on these RBAC objects, similar to how it was done in https://github.com/rook/rook/blob/master/pkg/operator/api/api_test.go#L107 ?

Reply:
ok ill add this